tests/json-fixtures/test_state.py

```diff
@@ -137,24 +111,28 @@ def get_block_hash_for_testing(self, block_number):
 )
 
 
-ChainForTesting = Chain.configure(
-    name='ChainForTesting',
-    vm_configuration=(
-        (constants.GENESIS_BLOCK_NUMBER, FrontierVMForTesting),
-        (constants.HOMESTEAD_MAINNET_BLOCK, HomesteadVMForTesting),
-        (constants.EIP150_MAINNET_BLOCK, EIP150VMForTesting),
-    ),
-)
-
-
-def test_found_state_fixtures():
-    assert len(FIXTURES) != 0
+@pytest.fixture
+def fixture_vm_class(fixture_data):
+    _, _, fork_name, _ = fixture_data
```

Comment:
Oh, so it looks like the unpacking cannot be done where I'd mentioned it because for different tests there may be a different number of items in fixture_data?

Reply:
Yeah, the statetest fixtures end up unpacking to 4 items since they have different outcomes for each fork and different inputs and outputs within each fork

everything else is just `fixture_path` and `fixture_key`.